etc/config/ispc.amazon.properties

```diff
@@ -1,6 +1,8 @@
-compilers=ispc191
-defaultCompiler=ispc191
-compilerType=ispc
```

Comment:
Did you mean to remove the `compilerType` in this change? I think it's probably necessary?

Reply:
I think it's redundant, it's it inherited from ispc.defaults.properties?

Reply:
Oh you're probably right! Thanks for clarifying :)